### PR TITLE
Enterprise Java Microsite

### PR DESCRIPTION
**Pull request by myfear**

### CODE REVIEW

File: enterprise-java.html.slim [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      asciidoc:   
+		Find all you need in the link:http://www.jboss.org/products/eap/get-started/[Getting Started Secion] on JBoss developer. It's not much more than just hitting download and unzip into a folder of your choice. And we also provide link:http://www.jboss.org/products/eap/developer-materials[Developer Materials] and link:http://www.redhat.com/en/services/training[Trainings] to improve your skills, or the skills of your team, with our hands-on, role-based training for IT professionals and developers. If you feel the need for latest innovation and specifications, you need to look at link:http://wildfly.org/[WildFly]. This is the community version and helps you build flexible and lightweight applications.
+    .large-6.column
+	h4 Learn About Technology	
+	asciidoc:     
+		There are many resources available for link:http//:www[Red Hat JBoss Enterprise Application Platform] on link:http://www.jboss.org[JBoss Developer] or the link:http://access.redhat.com[Red Hat Customer Portal]. You can also follow our Developer Advocates link:http://www.arungupta.me[Arun Gupta] or link://blog.eisele.net[Markus Eisele] and see, what they have to share.
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: Broken link.

File: enterprise-java.html.slim [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    .large-6.column
+      h4 Learn About Community
+      asciidoc:  
+		 Work alongside smart, motivated people to apply your skills toward a bigger effort. Look at the link:http://javaee-samples.github.io/[Java EE 7 Samples Project] or contribute to link:https://github.com/jboss-developer/www.jboss.org[JBoss Developer] and don't forget to find solutions in our link:http://www.jboss.org/forums/[Forums].
+    .large-12.column
+      h4 Getting Started Is Easy
+      asciidoc:   
+		Find all you need in the link:http://www.jboss.org/products/eap/get-started/[Getting Started Secion] on JBoss developer. It's not much more than just hitting download and unzip into a folder of your choice. And we also provide link:http://www.jboss.org/products/eap/developer-materials[Developer Materials] and link:http://www.redhat.com/en/services/training[Trainings] to improve your skills, or the skills of your team, with our hands-on, role-based training for IT professionals and developers. If you feel the need for latest innovation and specifications, you need to look at link:http://wildfly.org/[WildFly]. This is the community version and helps you build flexible and lightweight applications.
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: Secion  -> Section

Comment (paulrobinson) [inline enterprise-java.html.slim]: Trainings -> Training

File: enterprise-java.html.slim [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	asciidoc:     
+		There are many resources available for link:http//:www[Red Hat JBoss Enterprise Application Platform] on link:http://www.jboss.org[JBoss Developer] or the link:http://access.redhat.com[Red Hat Customer Portal]. You can also follow our Developer Advocates link:http://www.arungupta.me[Arun Gupta] or link://blog.eisele.net[Markus Eisele] and see, what they have to share.
+
+.wide.microsite-large-icons
+  .row
+    .large-4.column.center.microsite-large-icon
+      i.fa.fa-map-marker
+    .large-20.column
+      h3 Supported Middleware vs. Upstream Projects
+      p Our supported product portfolio provides enterprise-ready software built from an innovative community of projects. It delivers enterprise-class stability, enterprise-class support SLAs, guaranteed patches, updates and hot-fixes, and legal assurance. We achieve this by challenge convention and redefine Enterprise Java through community-driven projects. Community projects represent the latest technologies for use in cutting-edge applications and offer best-effort, community support.
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: I think you mean:

    We achieve this by challenging convention and redefining Enterprise Java through community-driven projects

File: enterprise-java.html.slim [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      p Our supported product portfolio provides enterprise-ready software built from an innovative community of projects. It delivers enterprise-class stability, enterprise-class support SLAs, guaranteed patches, updates and hot-fixes, and legal assurance. We achieve this by challenging convention and redefining Enterprise Java through community-driven projects. Community projects represent the latest technologies for use in cutting-edge applications and offer best-effort, community support.
+	a.button(href="http://www.jboss.org/technology/") Learn More
+
+.row	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=xa_gtRDpwyQ"])	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=kEYVBtkjAE4"])	
+	.large-8.columns
+		= video_player(site.videos["https://www.youtube.com/v=cMnPrEiYez0"])
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: site.videos is a collection of videos that are indexed by the site. They are keyed on the url. For this to work, the URL needs to represent a video already indexed. You can follow these instructions: https://github.com/jboss-developer/www.jboss.org/blob/master/CONTRIBUTING.md#to-add-a-youtube-video-to-the-site

Then you either need to wait for the cache to clear. This takes upto 6hrs, or I can force it for you.

Comment (paulrobinson) [inline enterprise-java.html.slim]: I also get this when visiting the links: https://www.youtube.com/v=xa_gtRDpwyQ

**Comment by myfear** [comment]
@paulrobinson can you re-test this. The site looks ok, the video cache wasn't updated when I pushed in the changes.

**Comment by paulrobinson** [comment]
retest this please

**Comment by paulrobinson** [comment]
BTW, you can trigger a retest, by commenting as I did above.

**Comment by paulrobinson** [comment]
The banner is tiling at my resolution: https://www.dropbox.com/s/8m425pl7gemvih5/Screenshot%202015-02-25%2011.22.45.png?dl=0

I'll ask James about what can be done here.

**Comment by paulrobinson** [comment]
Also, the text is hard to read on this background (IMO): https://www.dropbox.com/s/abcchdzcaftcv7n/Screenshot%202015-02-25%2011.23.16.png?dl=0. Maybe we need a slightly different banner. One which tiles, and is easier to display text over?

**Comment by paulrobinson** [comment]
@myfear Where did the 'We Do All The Enterprise Java' title come from? It seems a bit strange to me. I'm not yet sure what I would suggest as an alternative.

**Comment by paulrobinson** [comment]
It would be good if we could get the videos lined up better: https://www.dropbox.com/s/i95nvxdlj7nubm2/Screenshot%202015-02-25%2011.27.08.png?dl=0

However, I appreciate that you have no control over this, due to the use of the 'video_player' widget. Maybe switching the left and middle would look better: https://www.dropbox.com/s/rd9mlprcopx2lia/Screenshot%202015-02-25%2011.30.53.png?dl=0

**Comment by myfear** [comment]
The title was my idea. No source except me. 
Should express the general approach to redhat JBoss Middleware. 
Video alignment is an issue. I don't have a way to solve it. Changing the order of the videos later. 
Regarding the hero image,  James is on it.

**Comment by paulrobinson** [comment]
What about "The Place to Come for Enterprise Java"?

**Comment by myfear** [comment]
retest this please

File: enterprise-java.html.slim [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+layout: microsite
+hero_class: enterprise-java
+---
+
+= define 'hero'
+  .row
+    .large-13.column
+      h2 The Place to Come for Enterprise Java
+    .large-8.column
+      p JBoss® Enterprise Application Platform 6 provides an innovative modular, cloud-ready architecture, powerful management and automation, and world class developer productivity. It is Java™ EE 6 certified and features powerful yet flexible management, improved performance and scalability, and many new features to maximize developer productivity. It builds on the community foundation of WildFly.
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: First usage, so needs to be: Red Hat JBoss Enterprise Application Platform

File: enterprise-java.html.slim [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+		Find all you need in the link:http://www.jboss.org/products/eap/get-started/[Getting Started Section] on JBoss developer. It's not much more than just hitting download and unzip into a folder of your choice. And we also provide link:http://www.jboss.org/products/eap/developer-materials[Developer Materials] and link:http://www.redhat.com/en/services/training[Training] to improve your skills, or the skills of your team, with our hands-on, role-based training for IT professionals and developers. If you feel the need for latest innovation and specifications, you need to look at link:http://wildfly.org/[WildFly]. This is the community version and helps you build flexible and lightweight applications.
+    .large-6.column
+	h4 Learn About Technology	
+	asciidoc:     
+		There are many resources available for link:http//:www.jboss.org/products/eap/overview/[Red Hat JBoss Enterprise Application Platform] on link:http://www.jboss.org[JBoss Developer] or the link:http://access.redhat.com[Red Hat Customer Portal]. You can also follow our Developer Advocates link:http://www.arungupta.me[Arun Gupta] or link://blog.eisele.net[Markus Eisele] and see, what they have to share.
```
Comment (paulrobinson) [inline enterprise-java.html.slim]: You can abbreviate "Red Hat JBoss Enterprise Application Platform" to "JBoss EAP" if you want.

**Comment by paulrobinson** [comment]
I think this is ready now? If so, can you squash the commits, then i will merge.

**Comment by myfear** [comment]
Squashed and ready to merge.

**Comment by paulrobinson** [comment]
retest this please

**Comment by myfear** [comment]
@paulrobinson any idea why this is failing? The site looks good to me.

**Comment by paulrobinson** [comment]
retest this please

**Pull request merged**